### ISSUE TITLE
How to save and load the autonet model?

### ISSUE THREAD

**Issue by maxmarketit** [automl/Auto-PyTorch#53]
Related to https://github.com/automl/Auto-PyTorch/issues/20

But I figure that `torch.save(model2 = autonet.get_pytorch_model(), 'model.pth')` does not save preprocessing. And I find that using `torch.save` and `torch.load` would not guarantee the same result from 
`autonet.predict` because preprocessing is missing.

So how can I save the `autonet` model and load or How can I find out the preprocessing auto-pytorch found and save and load the same procedure? 

ps. I am using `develop` branch.

**Comment by ilmarivikstrom** [comment]
I also have been wondering about this for a while now. I found out that an exported model does not perform as well as you would expect. What is the workaround for this?

**Comment by LMZimmer** [comment]
As pointed out correctly, Auto-PyTorch currently only offers saving the model and not the preprocessing. The only work around right now is to refit the autonet with the incumbent config. We will extend this functionality in coming releases.

**Comment by maxmarketit** [comment]
I think we can do with something like
```
autonet = AutoNetRegression(
    preprocessors = ['none'],  
    normalization_strategies = ['none'] 
)
```
I saw the results are the same for `torch.save`ed model and the `autonet.predict` for my case.

I am not sure about how the preprocessing or normalization goes but if it is just
`sklearn.preprocessing` functions, we think we can take them apart and save the actual function
applied to and the configuration. 

Could you give me some direction on this?

**Comment by franchuterivera** [comment]
Hello, with the new refactored version of the code, available in the development branch, we have documented how one can save and load an auto-pytorch estimator. This can be seen [here](https://automl.github.io/Auto-PyTorch/development/examples/40_advanced/example_visualization.html#sphx-glr-examples-40-advanced-example-visualization-py).

I am closing this old issue and if there are further questions, please let us know. We will be happy to help.
